salt/states/boto3_route53.py

```diff
@@ -595,54 +595,55 @@ def rr_present(name, HostedZoneId=None, DomainName=None, PrivateZone=False, Name
 
     # Convert any magic RR values to something AWS will understand, and otherwise clean them up.
     fixed_rrs = []
-    for rr in ResourceRecords:
-        if rr.startswith('magic:'):
-            fields = rr.split(':')
-            if fields[1] == 'ec2_instance_tag':
-                if len(fields) != 5:
-                    log.warning("Invalid magic RR value seen: '%s'.  Passing as-is.", rr)
-                    fixed_rrs += [rr]
-                    continue
-                tag_name = fields[2]
-                tag_value = fields[3]
-                instance_attr = fields[4]
-                good_states = ('pending', 'rebooting', 'running', 'stopping', 'stopped')
-                r = __salt__['boto_ec2.find_instances'](
-                        tags={tag_name: tag_value}, return_objs=True, in_states=good_states,
-                        region=region, key=key, keyid=keyid, profile=profile)
-                if len(r) < 1:
-                    ret['comment'] = 'No EC2 instance with tag {} == {} found'.format(tag_name,
-                            tag_value)
-                    log.error(ret['comment'])
-                    ret['result'] = False
-                    return ret
-                if len(r) > 1:
-                    ret['comment'] = 'Multiple EC2 instances with tag {} == {} found'.format(
-                            tag_name, tag_value)
-                    log.error(ret['comment'])
-                    ret['result'] = False
-                    return ret
-                instance = r[0]
-                res = getattr(instance, instance_attr, None)
-                if res:
-                    log.debug('Found %s %s for instance %s', instance_attr, res, instance.id)
-                    fixed_rrs += [_to_aws_encoding(res)]
+    if ResourceRecords:
+        for rr in ResourceRecords:
+            if rr.startswith('magic:'):
+                fields = rr.split(':')
+                if fields[1] == 'ec2_instance_tag':
+                    if len(fields) != 5:
+                        log.warning("Invalid magic RR value seen: '{}'.  Passing as-is.".format(rr))
```

Comment:
Can you switch this back to using `%s` notation instead of `.format()`? This will break the unicode changes we've been making recently.